docker-compose.yml

```diff
@@ -9,21 +9,32 @@ services:
       - weblate-data:/app/data
     env_file:
       - ./environment
+    ports:
+      - "8000:8000"
     restart: always
     depends_on:
       - database
       - cache
   web:
-    build: nginx
-    links:
-      - weblate
+    image: jwilder/nginx-proxy
+    volumes:
+      - /var/run/docker.sock:/tmp/docker.sock:ro
+      - ./certs:/etc/nginx/certs:ro
+      - /etc/nginx/vhost.d
+      - /usr/share/nginx/html
     ports:
       - "80:80"
-    volumes_from:
-      - weblate:ro
```

Comment:
I've documented what should be IMHO done with the container in https://github.com/WeblateOrg/docker/issues/50, the only reason I've not yet done that is lack of time.

I'm not saying https support is not necessary, but I don't think it makes sense to support it at this level. IMHO much better approach is to have separate docker container serving as endpoint for https and dispatching request to others. On the other side the Docker world is probably get used to duplicating same functionality in all containers rather than reusing smaller modules...